{{cookiecutter.github_repository}}/provisioner/roles/postgresql/tasks/main.yml

```diff
@@ -11,20 +11,17 @@
   shell: wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
 
 - name: Add to PostgreSQL.list
-  shell: sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -sc)-pgdg main" > /etc/apt/sources.list.d/PostgreSQL.list'
+  shell: sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -sc)-pgdg main" > /etc/apt/sources.list.d/PostgreSQL.list'
```

Comment:
How about using https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html instead of raw cli command.